### ISSUE TITLE
[src/library_glfw.js] onDrop event not processed

### ISSUE THREAD

**Issue by raysan5** [emscripten-core/emscripten#10004]
Just noted that files drag and drop functionality stop working for [raylib](https://github.com/raysan5/raylib) when compiling for web with emscripten. raylib uses GLFW3 backend for window/input events management.

I compiled my [core_drop_files.c](https://github.com/raysan5/raylib.com/blob/master/examples/web/core/core_drop_files.c) example with `-s FORCE_FILESYSTEM=1` to allow file drag an drop...
```
emcc -o core/core_drop_files.html core/core_drop_files.c -Wall -s USE_GLFW=3 --shell-file C:\GitHub\raylib\src\shell.html -I. -IC:\GitHub\raylib/src -IC:\GitHub\raylib/src/external -L. -LC:\GitHub\raylib/src -LC:\GitHub\raylib/src C:\GitHub\raylib/src/libraylib.bc -DPLATFORM_WEB -s TOTAL_MEMORY=67108864 -s FORCE_FILESYSTEM=1
```
...but it seems the following line on `library_glfw.js`
```
#if '$FS' in addedLibraryItems
```
always evaluates to false. Commenting that line (and the closing `#endif`) everything works as expected.

I tracked the issue to this commit: https://github.com/emscripten-core/emscripten/commit/7cae0636bbe44092b20b3a7679ed4562e31a0b89#diff-ea65b4a414f7c8ccd91dfc0279b0f90c where that `#if` guard was added.

As per my understanding, that guard should be evaluated to true when `-s FORCE_FILESYSTEM=1`.

Also noted that `.glfw_dropped_files` route is added to dropped files, it would be nice to be able to define that route if desired... but not a big deal.

**Comment by kripken** [comment]
Thanks @raysan5 ! Yes, this code was wrong. Fix in #10058.

It would be good to add a test for this, so it doesn't break again. Not sure how to test drag and drop automatically, though, but probably there's a way? A PR adding a test would be very welcome if you or someone else has time.

**Comment by raysan5** [comment]
Hi @kripken! Thanks for the review! :D 

I neither know how to test the event automatically... actually it took me a while to find the issue because I tried `tests/glfw3.c` and callback was working... in any case, I'll think about it.
